test/e2e/proxy_test.go

```diff
@@ -68,7 +67,7 @@ func TestProxyServiceSessionAffinity(t *testing.T) {
 	svc, err := data.createNginxClusterIPService(true)
 	require.NoError(t, err)
 	ingressIPs := []string{"169.254.169.253", "169.254.169.254"}
-	_, err = data.createNginxLoadBalancerService(false, ingressIPs)
+	_, err = data.createNginxLoadBalancerService(true, ingressIPs)
```

Comment:
how was the test working previously if session affinity wasn't enabled?

Reply:
actually I assume the test was broken and we didn't detect it because we do not enable AntreaProxy for Jenkins tests, except for the Windows CI, for which we do not run the Antrea e2e tests. We also do not run these tests for cloud CI jobs.

Reply:
Correct, this test was just missed in all of our current checks. We skipped this in KIND and didn't run in Jenkins, also we don't run e2e for windows.